### ISSUE TITLE
Cannot bind argument to parameter 'ChangeSet' because it is null.

### ISSUE THREAD

**Issue by FallenHoot** [Azure/AzOps#341]
**Describe the bug**

![image](https://user-images.githubusercontent.com/19664186/118323549-16f9da80-b501-11eb-9de0-43c1ae289ff2.png)

**Steps to reproduce**
I created a feature branch
Copied MG Sandbox renamed the copy to **test** and changed the .json to reflect **test**
Pushed code to feature branch
Did a PR to main branch
AzOps - Push triggered
After about 2:30 min it errors

**Testing**
I ran the setup like it does in the .yml file on my local machine.
![image](https://user-images.githubusercontent.com/19664186/118323904-98516d00-b501-11eb-81cc-4dfde501e19e.png)

It didn't work, because it has no clue what it is doing, but it doesn't complain about _"Cannot bind
     | argument to parameter 'ChangeSet' because it is null."_

**Comment by FallenHoot** [comment]
I wanted to verify that the ```$diff``` was not NULL or Whitespaced

I added the following the the .yml
```If ([String]::IsNullOrWhiteSpace((Get-content /tmp/diff.txt))){ Write-Host "File is empty"; } Else {$diff = Get-Content -Path /tmp/diff.txt}```

The file is empty... 

![image](https://user-images.githubusercontent.com/19664186/118331900-8a9fe580-b509-11eb-80f2-71408149efcb.png)

**Comment by FallenHoot** [comment]
I have now added some error handling into the push.yml for GitHub Actions
![image](https://user-images.githubusercontent.com/19664186/118356536-06d01280-b576-11eb-9c6e-3da3c6cf157f.png)

**What I have noticed is two things.** 
It states the file is empty sometimes and sometimes it has data. What I have done is created the Diff on my local machine and pushed it to github, this works, but then it doesn't actually do anything. It doesn't add the new MG.   I have test about 23 times, and nothing. 

For some reason it doesn't create the diff.txt
